package.json

```diff
@@ -21,6 +21,7 @@
   "author": "Kyber SC Team",
   "license": "MIT",
   "dependencies": {
+    "@kyber.network/utils-sc": "^2.0.0",
```

Comment:
I think we should fix this.
but later it should be 2.1.0 after Anton is done.

Reply:
Yes, will wait for Anton to be done.